pkg/utils/utils.go

```diff
@@ -18,3 +20,14 @@ func GetBytes(key interface{}) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func ParseURL(s string) (*url.URL, error) {
+	if strings.HasPrefix(s, "bolt://") {
```

Comment:
heh, i was wondering how you were going to do windows paths with this. the other option is to do the file:// trick browsers do. isn't "C:\Windows\SYSTEM\cool.dll" translated to "file://C|/Windows/SYSTEM/cool.dll" or something? it's been a long time but i vaguely remember that. but this is fine

Reply:
The ParseURL part was somewhat easier solution, because then I don't have to find all the different places where filepath.Join, flags are passed or read from some configuration. (Although, not sure about the func name, there might be something better).

Reply:
Ah, also forgot, that this accidentally fixed some things on unix paths as well.

````
u, _ := url.Parse("bolt://.storj/sub/example.db")
fmt.Println("host", u.Host) // output: .storj
fmt.Println("path", u.Path) // output: /sub/example.db
```